sps-team-38/src/main/webapp/script.js

```diff
@@ -53,26 +53,44 @@ function addInput(divName){
     }
 }   
 
+//Form object to store locally, a form retrieved from database.
+function Form(title, capacity, postDate, expiryDate){
+    this.title = title;
+    this.capacity = capacity;
+    this.postDate = postDate;
+    this.expiryDate = expiryDate
+}
+
 const formListElement = document.getElementById('form-list');
 const list = document.querySelector('#form-list ul');
+
   // Load existing forms in database
 function loadForms(){
-    fetch('/list-forms').then(response => response.json()).then((forms) => {
+    // fetch('/list-forms').then(response => response.json()).then((forms) => {
+    // forms.forEach((form) => {
+    //     formListElement.appendChild(renderForm(form));
+    // })
+    // });
```

Comment:
If this isn't being used I don't think it should be in the PR either to avoid confusion down the line (just think that's best practice- my PA from Spring session always stopped me when I was going to do this)

Reply:
Got it. Will remove.